tests/e2e/e2e_ibc_test.go

```diff
@@ -164,6 +166,14 @@ func (s *E2ETest) TestIBCTokenTransfer() {
 		// supply don't change
 		s.checkSupply(gaiaAPIEndpoint, umeeIBCHash, math.ZeroInt())
 
+		// << Recevier Addr = maximum length + 1
+		// send $110 UMEE from umee to gaia (token_quota is 100$)
+		recvAddr := sdkutil.GenerateString(ibcutil.MaximumMemoLength + 1)
+		s.SendIBC(s.Chain.ID, setup.GaiaChainID, recvAddr, exceedUmee, true, "", "")
+		// check the ibc (umee) quota after ibc txs - this one should have failed
+		// supply don't change
+		s.checkSupply(gaiaAPIEndpoint, umeeIBCHash, math.ZeroInt())
```

Comment:
I think it would be better to do unit test, this way we don't increase the time of already long e2e runs